setup.py

```diff
@@ -225,6 +225,9 @@
         'invenio_db.models': [
             'inspire_workflows_audit = inspirehep.modules.workflows.models',
         ],
+        'invenio_db.alembic': [
+            'inspirehep = inspirehep:alembic',
+        ],
```

Comment:
Let's keep our entry points alphabetically sorted! This should go before `invenio_db.models`.